WordPress/Classes/ViewRelated/Reader/ReaderRelativeTimeFormatter.swift

```diff
@@ -0,0 +1,74 @@
+import Foundation
+
+class ReaderRelativeTimeFormatter: NSObject {
```

Comment:
I don't think this is coupled to Reader, `RelativeTimeFormatter` would be ok. :)

Reply:
I thought a lot about the naming of the file and I should have mentioned it in the description. 

So, while this is a general "relative"-ish formatter it currently contains logic very specific to the reader, there's no configuration options, and defaults to displaying "just now" for any potential future dates. 

I don't want to confuse anyone in the project who sees `RelativeTimeFormatter` to think it's going to work like `NSRelativeDateTimeFormatter` or `TTTimeIntervalFormatter` 

If this logic is going to be applied in other locations in the app I can change this. 

So, at this point I'd think it's best to keep it relative to the reader. Thoughts?

Reply:
Agree